runtime/protocol/http/common/src/aws/smithy/kotlin/runtime/http/endpoints/util/Util.kt

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+package aws.smithy.kotlin.runtime.http.endpoints.util
+
+import aws.smithy.kotlin.runtime.util.InternalApi
+import aws.smithy.kotlin.runtime.util.net.Host
+import aws.smithy.kotlin.runtime.util.net.isIpv6
+import aws.smithy.kotlin.runtime.util.net.isValidHostname
+import aws.smithy.kotlin.runtime.util.text.ensureSuffix
+import aws.smithy.kotlin.runtime.util.text.urlEncodeComponent
+
+@InternalApi
+public fun isSet(value: Any?): Boolean = value != null
```

Comment:
I'm assuming these are endpoint stdlib functions we are providing? 

If so I'd suggest adding some documentation to that effect so that we can at least mentally grok this belongs to an SEP. I'd also suggest  moving these out of a generic `Util` and into something like `Runtime` or `Functions` or whatever makes sense to match SEP verbiage.